plugins/plugin-sass/README.md

```diff
@@ -0,0 +1,23 @@
+# @snowpack/plugin-sass
+
+This plugin adds [Sass](https://sass-lang.com/) support to any Snowpack project. With this plugin, you can import any  `*.scss` Sass from JavaScript and have it compile to CSS.
+
+This plugin also supports `.module.scss` Sass modules. [Learn more.](https://www.snowpack.dev/#import-css-modules)
+
+## Usage
+
+```bash
+npm i @snowpack/plugin-sass
```

Comment:
```suggestion
npm i sass @snowpack/plugin-sass
```

Reply:
great call, I need to add instructions for this. 

Part of what I tried to do was let you provide the Sass CLI yourself, meaning you could use either Dart-Sass or Node-Sass. `npm i sass` would force node-sass, which I know you've had mixed experience with. Thoughts on providing two different snippets here, one for dart (recommended) and one for node-sass (not as good, but may be easier).

Reply:
Nope! `sass` is Dart Sass (the official version) 🙂. A lot of packages use `node-sass` under-the-hood, which is what I’d recommend avoiding

Reply:
(edited the above) Also, feel free to ignore this if you have a more comprehensive plan with config. Only trying to provide recommendations—as long as there’s a happy path that involves little config, I’m happy

Reply:
ah yea, you're right. I thought they shared the same PATH executable.

Unfortunately, the APIs aren't an exact match, either.  sass-loader has to do some intense stuff to support both APIs: https://github.com/webpack-contrib/sass-loader#implementation

Given the plugin API changes already in this PR, I'd like to ship this PR with just `sass` support for now. Then, leave `node-sass` support for a follow-up. How does that sound?

Reply:
Huh. looks like they even went ahead and marked node-sass as deprecated (or at least, are talking about doing it)

https://github.com/sass/node-sass/issues/2952

Reply:
That works for me. I don’t think Dart Sass has any downsides; my understanding is node-sass is only preferable when doing programmatic things in Node. Which doesn’t seem possible with this plugin anyway? Honestly I’d be fine just not supporting node-sass in general

Reply:
Yea, see my comment above, leaning this way as well